test/rubygems/test_gem_commands_build_command.rb

```diff
@@ -389,7 +389,7 @@ def test_build_auto_resign_cert
     output = @ui.output.split "\n"
     assert_equal "INFO:  Your certificate has expired, trying to re-sign it...", output.shift
     assert_equal "INFO:  Your cert: #{tmp_expired_cert_file } has been auto re-signed with the key: #{tmp_private_key_file}", output.shift
-    assert_match /INFO:  Your expired cert will be located at: .+\Wgem-public_cert\.pem\.expired\.[0-9]+/, output.shift
+    assert_match(/INFO:  Your expired cert will be located at: .+\Wgem-public_cert\.pem\.expired\.[0-9]+/, output.shift)
```

Comment:
Why this change in particular?

Reply:
Because it's what the warning is telling me to do... Otherwise the code is ambigous because `/` could also mean integer division... I guess? :thinking:. Not really sure where the ambiguity lies, I just know you fix the warning by adding parentheses :laughing:

Reply:
Ah i see, sorry i didn't read well the description of the issue 😂